### DISCUSSION TITLE
Possibility to search in description

### DISCUSSION THREAD

**Discussion by dwaldhans** [ananthakumaran/paisa#219] [Q&A] [answered]
Hello,

According to https://paisa.fyi/reference/bulk-edit/ it is only possible to search in comments/notes and not in the description (the text to the right of the transaction date).

Am I missing something or is this really not possible?

Kind regards,
David

**Comment by ananthakumaran** [comment, marked as answer]
try payee

```
payee =~ /uber/i
payee = "Advance Tax"
```

**Reply by dwaldhans** [reply]
Ah, that's it, thank you! 😃
